components/places/src/storage/bookmarks.rs

```diff
@@ -1124,7 +1124,10 @@ fn inflate(
 
 /// Fetch the tree starting at the specified folder guid.
 /// Returns a BookmarkTreeNode::Folder(_)
```

Comment:
Ah, one more thing, this should be updated.
```suggestion
/// Returns a `BookmarkTreeNode::Folder(_)`, its parent's guid (if any), and
/// position inside its parent.
```

Reply:
Please re-review whenever you get a chance, thanks!